kythe-facts-verifier.sh

```diff
@@ -24,11 +24,15 @@ EOF
 set -o pipefail
 KYTHE_BINDIR="/opt/kythe/tools"
 KYTHE_OUT="./kythe-out"
+VERILOG_TEST_FILES="./verilog/tools/kythe/testdata/more_testdata/*.sv"
```

Comment:
Perhaps comment that each of these are expected to be self-contained single-file test cases.